composer.json

```diff
@@ -28,7 +28,7 @@
   },
   "require": {
     "composer/installers": "~1.0",
-    "php": ">=5.6|^7|^8"
+    "php": ">=7|^8"
```

Comment:
What about `"phpunit/phpunit": "^4|^5|^6|^7|^8|^9",`?

Reply:
PHPUnit is installed by `yoast/phpunit-polyfills`. This line should be removed. CC @swissspidy

Reply:
Indeed, it should be safe to simply remove that `phpunit/phpunit` line (and run `composer update` afterwards) because it's required by `yoast/phpunit-polyfills` anyway.

Reply:
Removed in https://github.com/WordPress/performance/pull/851/commits/b5179cfbcaba41ef327436d0513a370699be7b7f. I don't know why `composer update` did not solve the CI job. Could anyone guide me.